pkgs/applications/networking/p2p/zeronet/default.nix

```diff
@@ -1,19 +1,25 @@
-{ stdenv, fetchFromGitHub, python2Packages }:
+{ stdenv, fetchFromGitHub, python3Packages }:
 
-python2Packages.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "zeronet";
-  version = "0.6.5";
+  version = "0.7.1";
   format = "other";
 
   src = fetchFromGitHub {
     owner = "HelloZeroNet";
     repo = "ZeroNet";
     rev = "v${version}";
-    sha256 = "1b9jg525jq40czyw9xh9ck90fl9rzrdmzl2mzhqqhz23idb60j4a";
+    sha256 = "04prgicm0yjh2klcxdgwx1mvlsxxi2bdkzfcvysvixbgq20wjvdk";
   };
 
-  propagatedBuildInputs = with python2Packages; [ msgpack gevent ];
-  buildPhase = "${python2Packages.python.interpreter} -O -m compileall .";
+  propagatedBuildInputs = with python3Packages; [
+    gevent msgpack base58 merkletools rsa pysocks pyasn1 websocket_client
+    gevent-websocket rencode bitcoinlib maxminddb pyopenssl
```

Comment:
While we're at it and since there are so many now, could you put each input on a separate line?

Reply:
Is this required?
I have other packages that use this style, and to me, this would be introducing unneeded whitespace with no real bonus to clarification.

Reply:
Well, mostly it makes diffs cleaner. I _think_ it's also the preferred style in nixpkgs, but don't take my word for that.

> Is this required?

nah, it's no big deal

Reply:
I'll look in the docs I guess and see if there's a style guide or something like that, if so I'll change going forward.